### ISSUE TITLE
Use Data BC Catalog API

### ISSUE THREAD

**Issue by smnorris** [smnorris/bcdata#25]
Use the catalog [API](https://catalogue.data.gov.bc.ca/dataset/bc-data-catalogue-api) to find the DWDS path rather than scraping the catalog page. Path to download should be an item in the 'Resources' list. For example: https://catalogue.data.gov.bc.ca/api/3/action/package_show?id=conservation-lands

As there are multiple resources in the list I'm not quite sure how to pick the right one (in case the best download source isn't DWDS). Some examples would have to be checked out.

**Comment by smnorris** [comment]
Using the catalog api, it seems possible to tweak bcdata into something like a package manager.

Using [homebrew](http://brew.sh/) as a guide,
```
brew --help

Example usage:
  brew search [TEXT|/REGEX/]
  brew (info|home|options) [FORMULA...]
  brew install FORMULA...
  brew update
  brew upgrade [FORMULA...]
  brew uninstall FORMULA...
  brew list [FORMULA...]
```

bcdata cli could look something like this:
```
bcdata --help

Example usage:
  bcdata search [TEXT|/REGEX/]
  bcdata (info|home) [FORMULA...]
  bcdata get FORMULA...
  bcdata update
  bcdata refresh [FORMULA...]
  bcdata remove FORMULA...
```

**Comment by smnorris** [comment]
fixed with 2046eb87ef169c2e48c9c80bf074ce360cc6c952 but consider adding more interfaces to the API.

- package list command
- expand on `bcdata info` to include more results from `package_show` request
